### PR TITLE
Update French.properties

### PR DESCRIPTION
**Pull request by Ouaz**

### CODE REVIEW

**Review by HydrolienF — COMMENTED** [review]
Everything look find for me. Exept a "word:" that should be in french "mot :"

File: android/assets/jsons/translations/French.properties [hunk 1]
```diff
@@ -1464,7 +1464,7 @@ Pillaging this improvement yields [stats] = Piller cet aménagement rapporte [st
 Pillaging this improvement yields approximately [stats] = Piller cet aménagement rapporte environ [stats]
 Needs removal of terrain features to be built = Nécessite de supprimer les caractéristiques de terrain pour être construit
 Unit type = Type d'unité
-Units: = Unités :
+Units: = Unités:
```
[the remark this answers is not in the document]
Comment (Ouaz) [reply android/assets/jsons/translations/French.properties:1467]: It's related to a sub-title in the civilopedia.

Usually, the colon is not in the translation file, but directly in the code, so I can't add the space before:

colon1
colon2

For this one, I can, but then there's an inconsistency:
colon3

AFAIK, `Units:` is used only here (as it is a new string, added very recently).

Should I add back the space before the colon in french?

[the remark this answers is not in the document]
Comment (HydrolienF) [reply android/assets/jsons/translations/French.properties:1467]: 👍

**Comment by Ouaz** [comment]
> for very little gain.

Agreed.

The common rule (at least in french), is _word_ `thin space`**:**`regular space` _word_
For easy life, I use regular space before and after when translating. ^^

Though, in this particular case,  _word_ `no space`**:** is acceptable, as there's no word after the colon.

> Après tout, porquoi enfin sont les Français tellement fous? Le clavier perverti, gaspiller espace de nos textos avec trop du néant, etc etc

LMAO. True, French are so annoying, and french language is so overly complicated. :D

**Pull request merged**